release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2622,3 +2622,24 @@ link:https://access.redhat.com/solutions/6308491[{product-title} 4.8.10 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-8-11"]
+=== RHBA-2021:3429 - {product-title} 4.8.11 bug fix update
+
+Issued: 2021-09-14
+
+{product-title} release 4.8.11 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:3429[RHBA-2021:3429] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6324771[{product-title} 4.8.11 container image list]
+
+[id="ocp-4-8-11-bug-fixes"]
+==== Bug fixes
+
+* Previously, the `Serverless` add group did not contain `Event Sources` in the grouping. With this update, the `Serverless` add group has been updated to `Eventing` and `Event Sources` can now be found within the add group. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1999931[BZ#1999931])
```

Comment:
Maybe it's down to lack of familiarity on my part, but what is an add group? 

I spend very little time in the web UI--is this dev inside baseball terminology, or will the phrase make sense to readers?